app/components/SpeakingTimeContent.tsx

```diff
@@ -44,6 +47,19 @@ export default function SpeakingTimeContent({
     }
   };
   const state = useTransition();
+
+  // sort tags to ensure the "quick response" one is always the first in the array
+  tags.unshift(
+    tags.splice(
+      tags
+        .map(function (tag) {
+          return tag.sys.id;
+        })
+        .indexOf(QUICK_RESPONSE_TAG_ID),
+      1
+    )[0]
+  );
+
```

Comment:
I thought of doing this sorting in the `contentful.ts` file at the time of requesting the tags, but I wasn't sure about it. I prefer that here it's more connected to the functionality of the page than to the source of the data, but it's just gut feeling. If adding it to the parent component, I would then need to add it repeatedly to the 4 pages with the different languages :upside_down_face: which I prefer to avoid.

Any other idea @HerrBertling ?

Reply:
I‘d say this is the exact place where it should happen 😅👍🚀